front/js/cart.js

```diff
@@ -1,15 +1,278 @@
+        cartItemContentSettingsQuantityDiv.className = 'cart__item__content__settings__quantity';
+        cardItemContentSettingsDiv.appendChild(cartItemContentSettingsQuantityDiv);
+
+        let quantityProduct = document.createElement('p');
+        cartItemContentSettingsQuantityDiv.appendChild(quantityProduct);
+        quantityProduct.textContent = 'Qté : ';
+
+        let inputModifyProduct = document.createElement('input');
+        cartItemContentSettingsQuantityDiv.appendChild(inputModifyProduct);
+        inputModifyProduct.className = 'itemQuantity';
+        inputModifyProduct.setAttribute('type', 'number');
+        inputModifyProduct.setAttribute('name', 'itemQuantity');
+        inputModifyProduct.setAttribute('min', "1");
+        inputModifyProduct.setAttribute('max', "100");
+        inputModifyProduct.setAttribute('value', localStorageArea[item].quantityChoice);
+
+
+        // Au changement de la quantité du produit
+
+        inputModifyProduct.addEventListener('change', (event) => {
+
+            let resultQuantity = inputModifyProduct.value;
+
+            // On cherche le produit dont la valeur à changé
+
+            const productFound = localStorageArea.find(el => el.quantityChoice !== resultQuantity && el.productId == idProduct && el.colorChoice == color);
+
+            productFound.quantityChoice = resultQuantity;
+
+            localStorage.setItem('productStorage', JSON.stringify(localStorageArea));
+
+            return;
+
+        });
+
+
+        let cartItemContentSettingsDeleteDiv = document.createElement('div');
+        cardItemContentSettingsDiv.appendChild(cartItemContentSettingsDeleteDiv);
+        cartItemContentSettingsDeleteDiv.className = 'cart__item__content__settings__delete';
+
+        let deleteItem = document.createElement('p');
+        cartItemContentSettingsDeleteDiv.appendChild(deleteItem);
+        deleteItem.textContent = 'Supprimer';
+
+
+        // Au clic on suprime le produit dans le local storage
+
+        deleteItem.addEventListener('click', (event) => {
+
+            // On filtre les produits qui ne sont pas identique
+
+            localStorageArea = localStorageArea.filter(el => el.productId !== idProduct || el.colorChoice !== color);
+
+            // On sauvegarde le résultat dans le local storage
+
+            cartItemArticle.remove();
+
+            localStorage.setItem('productStorage', JSON.stringify(localStorageArea));
+
+            return;
+
+        });
 
-// Affichage des produits dans la page panier
 
-function displayCart() {
+    };
 
-    for (let product in productStorage) {
+};
 
+// Affichage du prix total du panier
 
+
+function totalCartPrice(productList) {
+
+    let total = 0;
+
+    for (let p = 0; p < productList.length; p++) {
+
+        let productPrice = productList[p].price;
+        let quantity = productStorageArea.quantityChoice;
+        total = quantity * productPrice;
+    };
+
+    console.log(total);
```

Comment:
Je pense que ça ne fonctionne pas ça.

Reply:
oui, en effet, je suis en work in progress sur ce sujet

Reply:
Yes je t'ai répondu normalement.